docs/docs-content/clusters/cluster-management/cluster-proxy.md

```diff
@@ -0,0 +1,223 @@
+  communications. For more information, refer to
+  [Enabled and Manage Proxy Configurations](../pcg/manage-pcg/configure-proxy.md).
+
+</TabItem>
+
+<TabItem value="Self-Hosted Palette Non-Edge">
+
+- A self-hosted Palette instance is deployed into an active and healthy Kubernetes cluster. Refer to
+  [Self-Hosted Palette Installation](../../enterprise-version/install-palette/install-palette.md) for additional
+  guidance.
+
+- The self-hosted Palette instance is configured to use the proxy server that you intend for your applications to use
+  for outbound communications.
+
+</TabItem>
+
+<TabItem value="Edge">
+
+- Your Edge host is configured to use the proxy server that you intend for your applications to use for outbound
+  communications. Refer to [Edge Host Installation](../edge/site-deployment/stage.md) for more information.
+
+</TabItem>
+
+</Tabs>
+
+## Configure Applications to Use Proxy Server
+
+<Tabs groupId="scenario">
+
+<TabItem value="Palette SaaS Non-Edge">
+
+1.  If you are using Palette SaaS, and your cluster does not have direct access to the internet, then the communication
+    between Palette SaaS and your cluster must go through a Private Cloud Gateway (PCG). For more information about
+    PCGs, refer to [Private Cloud Gateway](../pcg/pcg.md). You must configure proxy settings for the PCG, and then
+    clusters created by cloud accounts configured to use the PCG will inherit its proxy settings.
+
+    If you are provisioning the PCG using the Palette CLI, you can configure the proxy settings during the PCG
+    installation through the Palette CLI's interactive prompts. If you are using Helm to provision a PCG or have an
+    existing PCG that is not yet configured to use proxy, refer to
+    [Enable and Manage Proxy Configurations for PCG](../pcg/manage-pcg/configure-proxy.md) to learn how to install Reach
+    on a PCG cluster and use it to configure proxy settings.
+
+2.  Once you have configured proxy settings through Reach for your PCG, you can proceed to create your cluster using a
+    cloud account associated with the PCG. The cluster will inherit proxy settings from the PCG automatically.
+
+    However, you still need to specify which applications will use the proxy settings. You can do this by applying the
+    `spectrocloud.com/connection: proxy` label to the deployment, job, or daemon set in the pack that contains your
+    application.
+
+    You must apply the label to every specific job, deployment, or daemon set that needs to use the proxy servers. For
+    example, if you have a Kafka deployment that requires access to the internet through your proxy, you need to apply
+    the label to the Kafka deployment.
+
+    ```yaml {10}
+    apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: kafka
+    spec:
+      template:
+        metadata:
+          labels:
+            app: kafka
+            spectrocloud.com/connection: proxy
+    ```
```

Comment:
This part might benefit from geing its own step as it focuses on the cluster profile. That way we don;t overload step 2 as there is a lot going on there with the account registration and limitation of existing account not being able to use the PCG.